### ISSUE TITLE
question about the result.png in run/train and the result run by myself

### ISSUE THREAD

**Issue by tengxiao24** [Jacobi93/Alpha-IoU#9]
Dear author,
Thanks for your work. I have a question about the result.png in runs/train.The loss of objectness and val objectness  rise in the initial stage in voc_yolov5s_3diou result.png，I don't know how to explain this phenomenon.Besides,I cloned your code and run on my computer, i set batch-size=8 and set the funtcion bbox_iou function(DIoU=true , alpha=1) .And i got the mAP@0.5 = 75.04, mAP@0.5:0.95=47.95.
Could you please give me some suggestion on it.
Thanks.

**Comment by Jacobi93** [comment]
Thx for your interest in our project. 1. It is probable that the bbox regression loss and the classification loss is the training target so that the objectness may rise in initial stages, which normally drops soon in the following epochs. 2. I assume that your result is obtained by training yolov5s on PASCAL VOC, which is a little worse than the baseline. You should check your training settings, e.g., learning rate and batch size.

**Comment by tengxiao24** [comment]
Thanks for your reply. My training  settings are the same as yours except the batch-size,and my batch-size is 8. GPU is NVIDIA GeForce RTX 2060 SUPER 8G

**Comment by Jacobi93** [comment]
That should be the problem. As shown in Appendix B.1 Detailed Training Setup, we set the configuration for yolov5s following  "yolov5s.yaml" with batch size = 64, and the initial learning rate = 0.01. You should change the learning rate according to different batch sizes.

**Comment by tengxiao24** [comment]
Thanks,  i will try it.
